docs/source/config_file.rst

```diff
@@ -326,88 +412,36 @@ Miscellaneous strictness flags
    Prohibit equality checks, identity checks, and container checks between
    non-overlapping types.
 
-Platform configuration
-----------------------
-
-``always_true`` (comma-separated list of strings)
-    Specifies a list of variables that mypy will treat as
-    compile-time constants that are always true.
-
-``always_false`` (comma-separated list of strings)
-    Specifies a list of variables that mypy will treat as
-    compile-time constants that are always false.
-
-
-Global-only options
-*******************
-
-The following options may only be set in the global section (``[mypy]``).
-
-.. _config-file-import-discovery-global:
-
-Import discovery
-----------------
-
-For more information, see the :ref:`import discovery <import-discovery>`
-section of the command line docs.
-
-Note: this section describes only global-only import discovery options. See above for
-a list of import discovery options that may be used
-:ref:`both per-module and globally <config-file-import-discovery-per-module>`.
 
-``namespace_packages`` (bool, default False)
-    Enables :pep:`420` style namespace packages.  See :ref:`the
-    corresponding flag <import-discovery>` for more information.
-
-``python_executable`` (string)
-    Specifies the path to the Python executable to inspect to collect
-    a list of available :ref:`PEP 561 packages <installed-packages>`. User
-    home directory and environment variables will be expanded. Defaults to
-    the executable used to run mypy.
-
-``no_silence_site_packages`` (bool, default False)
-    Enables reporting error messages generated within :pep:`561` compliant packages.
-    Those error messages are suppressed by default, since you are usually
-    not able to control errors in 3rd party code.
-
-``mypy_path`` (string)
-    Specifies the paths to use, after trying the paths from ``MYPYPATH`` environment
-    variable.  Useful if you'd like to keep stubs in your repo, along with the config file.
-    Multiple paths are always separated with a ``:`` or ``,`` regardless of the platform.
-    User home directory and environment variables will be expanded.
+Configuring error messages
+**************************
```

Comment:
It looks like you dropped the blurb for this section. It was something like:

```
For more information, see the :ref:`configuring error messages <configuring-error-messages>`
section of the command line docs.
```

Reply:
Fixed.